### DISCUSSION TITLE
Make metadata data folders hidden

### DISCUSSION THREAD

**Discussion by ph00lt0** [ente-io/ente#613] [Enhancements]
Please make the metadata folders hidden by default. Personally I do show these folders anyway but for average non technical user I will look more polished. 

On unix (and macos) can be accomplished with adding . in front of folder name.

**Comment by vishnukvmd** [comment]
Hey @ph00lt0, thanks for the feedback, it makes sense to reduce the noise, will do this.

We're in the middle of building a CLI for exports, will keep this feedback in mind for that project as well!
